cmake/Platform/Linux/Install_linux.cmake

```diff
@@ -7,18 +7,20 @@
 #
 
 #! ly_install_code_function_override: Linux-specific copy function to handle RPATH fixes
-function(ly_install_code_function_override)
-    install(CODE
-"function(ly_copy source_file target_directory)
-    file(COPY \"\${source_file}\" DESTINATION \"\${target_directory}\" FILE_PERMISSIONS ${LY_COPY_PERMISSIONS} FOLLOW_SYMLINK_CHAIN)
-
-    get_filename_component(target_filename_ext \"\${source_file}\" LAST_EXT)
-    if(\"\${source_file}\" MATCHES \"qt/plugins\" AND \"\${target_filename_ext}\" STREQUAL \".so\")
-        get_filename_component(target_filename \"\${source_file}\" NAME)
-        file(RPATH_CHANGE FILE \"\${target_directory}/\${target_filename}\" OLD_RPATH \"\\$ORIGIN/../../lib\" NEW_RPATH \"\\$ORIGIN/..\")
+set(ly_copy_template [[
+function(ly_copy source_file target_directory)
+    file(COPY "${source_file}" DESTINATION "${target_directory}" FILE_PERMISSIONS @LY_COPY_PERMISSIONS@ FOLLOW_SYMLINK_CHAIN)
+    get_filename_component(target_filename_ext "${source_file}" LAST_EXT)
+    if("${source_file}" MATCHES "qt/plugins" AND "${target_filename_ext}" STREQUAL ".so")
+        get_filename_component(target_filename "${source_file}" NAME)
+        file(RPATH_CHANGE FILE "${target_directory}/${target_filename}" OLD_RPATH "\$ORIGIN/../../lib" NEW_RPATH "\$ORIGIN/..")
     endif()
-endfunction()"
-    )
+endfunction()]])
+
+string(CONFIGURE "${ly_copy_template}" ly_copy_function_linux @ONLY)
```

Comment:
The `string(CONFIGURE)` should be inside of the `ly_install_code_function_override` function to make sure that the LY_COPY_PERMISSIONS variable is set by that point.

With how it is now.
The configure call occurs when the the Install_linux.cmake file is included which happens early in the build process.

Reply:
Ok good call, will fix.  I looked at the output and it looked correct, but this seems safer.